esp/src/eclwatch/XrefFoundFilesWidget.js

```diff
@@ -70,6 +98,56 @@ define([
             return retVal;
         },
 
+        refreshActionState: function (event) {
+            var selection = this.grid.getSelected();
+            var hasSelection = selection.length;
+
+            registry.byId(this.id + "Delete").set("disabled", !hasSelection);
+            registry.byId(this.id + "Attach").set("disabled", !hasSelection);
+        },
+
+        _onDeleteFiles: function (event) {
+            var context = this;
+            var selections = this.grid.getSelected();
+            if (confirm(this.i18n.DeleteSelectedFiles)) {
+                var promises = [];
+                arrayUtil.forEach(selections, function (row, idx) {
+                    promises.push(WsDFUXref.DFUXRefArrayAction({
+                        request: {
+                            Cluster: context.params.Name,
+                            Type: 'Found',
+                            Action: 'Delete',
+                            XRefFiles_i2: row.Name
+                        }
+                    }));
+                });
+                all(promises).then(function() {
+                    context._onRefresh();
+                });
+            }
+        },
+
+        _onAttachFiles: function (event) {
+            var context = this;
+            var selections = this.grid.getSelected();
+            if (confirm(this.i18n.AddTheseFilesToDali)) {
```

Comment:
hould display a list of the items you are about to delete (similar to other places we delete)